### PR TITLE
Edit PersonBuilder.java

### PR DESCRIPTION
**Pull request by alex-setyawan**

### CODE REVIEW

File: src/test/java/seedu/address/testutil/PersonBuilder.java [hunk 1]
```diff
@@ -1,34 +1,63 @@
...
+    public static final String[] DEFAULT_BLOODTYPE = {"A", "POSITIVE"};
+    public static final String DEFAULT_COUNTRY = "Singapore";
+    // Medical history
+    public static final String DEFAULT_CONDITION = "High blood pressure";
+    public static final String DEFAULT_DOA = "2024-01-01";
+    public static final String DEFAULT_DIAGNOSIS = "Runny nose";
     public static final String DEFAULT_STATUS = "HEALTHY";
+    public static final String DEFAULT_SYMPTOM = "Sneezing, sniffing";
 
+    // Mandatory fields
```
Comment (laney0808) [inline src/test/java/seedu/address/testutil/PersonBuilder.java:43]: Address and status are mandatory fields

File: src/test/java/seedu/address/testutil/PersonBuilder.java [hunk 2]
```diff
@@ -83,8 +141,8 @@ public PersonBuilder withAddress(String address) {
     /**
      * Sets the {@code Phone} of the {@code Person} that we are building.
```
Comment (laney0808) [inline src/test/java/seedu/address/testutil/PersonBuilder.java:142]: Javadoc comments need to be edited and correspond to each method

**Review by laney0808 — COMMENTED** [review]
Overall good implementation. The only issues are with Javadoc comments and placement of the two compulsory fields (status and address) which are mistaken as optinal fields. No effect on the code though. Nice work!

**Review by jovantanyk — APPROVED** [review]
LGTM

**Pull request merged**